lib/wasix/tests/runners.rs

```diff
@@ -1,4 +1,6 @@
-#![cfg(feature = "webc_runner")]
+// Exclude runner tests from wasm targets for now, since they don't run properly
+// there.
+#![cfg(not(target_family = "wasm"))]
```

Comment:
I don't think we run the `wasmer-wasix` crate's unit tests on WebAssembly, do we?

Reply:
The problem is that the test fails to compile due to `TokioTaskManager` not being available on wasm targets.